### DISCUSSION TITLE
Workspaces / separate endpoints for different groups of datasets

### DISCUSSION THREAD

**Discussion by JakobMiksch** [geopython/pygeoapi#1325] [Q&A] [answered]
Does pygeoapi offer a concept of workspaces where I can have strictly separated endpoints for different groups of datasets?
I am inspired by the functionality GeoServer already has.

**Comment by francbartoli** [comment, marked as answer]
pygeoapi offers what the OGC API - Features specification provides per se. In this sense we do support [hierarchical collections](https://docs.pygeoapi.io/en/latest/configuration.html#hierarchical-collections) which seem to fit with your use case as far as I can understand.

**Reply by JakobMiksch** [reply]
Thanks. This was exactly what I was looking for 👍

**Reply by pvgenuchten** [reply]
on the other hand, i hope pygeoapi at some point can support dynamic config files, so you run a single instance of pygeoapi with multiple config files, based on the selected endpoint the relevant config file is used (similar approach as mapserver), or would it then be better to run multiple pygeoapi instances alongside each other (on different ports/containers)
